### ISSUE TITLE
After installing 

### ISSUE THREAD

**Issue by theking2** [Antergos/Cnchi#804]
### Problem:
Installing antergos with a swiss german keyboard results in a system that has the proper keyboard layout in the logon screen but on the desktop a german layout is used.

### Steps To Reproduce:
1.Start the installer and select German, Switzerland with legacy
2.Restart after install
3.Logon works with the Swiss german keyboard layout but the desktop is started with german layout
[cnchi.log](https://github.com/Antergos/Cnchi/files/1366223/cnchi.log)
[postinstall.log](https://github.com/Antergos/Cnchi/files/1366222/postinstall.log)

### Log Files
- Use a service like http://paste2.org/ to provide links to the logs.
- Link to /tmp/cnchi.log output.
- Link to /tmp/postinstall.log.

**Comment by karasu** [comment]
Thanks for reporting this! We will check it asap.

**Comment by killajoe** [comment]
Yes i already confirm this, it seems to be related to Gnome installation, as i can see this also on Live Environment, before cnchi is started (or i choose "try it") i have english (US) keyboard in settings, also if i choose german on boot menu, after installation there is also only en-US keybord setted inside new installed GNOME.

**Comment by killajoe** [comment]
and still persist in 17.11 for all DE/WM : https://forum.antergos.com/topic/8393/selected-keyboard-not-installed

**Comment by killajoe** [comment]
So i found what is wrong: you need to choose german, and then a variant, if you do not choose a variant , it uses US-default keyboard...
